tests/components/deconz/test_binary_sensor.py

```diff
@@ -76,22 +75,13 @@ async def test_binary_sensors(hass):
     data = deepcopy(DECONZ_WEB_REQUEST)
     data["sensors"] = deepcopy(SENSORS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
-    assert "binary_sensor.presence_sensor" in gateway.deconz_ids
-    assert "binary_sensor.temperature_sensor" not in gateway.deconz_ids
-    assert "binary_sensor.clip_presence_sensor" not in gateway.deconz_ids
-    assert "binary_sensor.vibration_sensor" in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 3
 
+    assert len(hass.states.async_all()) == 3
     presence_sensor = hass.states.get("binary_sensor.presence_sensor")
     assert presence_sensor.state == "off"
     assert presence_sensor.attributes["device_class"] == DEVICE_CLASS_MOTION
-
-    temperature_sensor = hass.states.get("binary_sensor.temperature_sensor")
-    assert temperature_sensor is None
-
-    clip_presence_sensor = hass.states.get("binary_sensor.clip_presence_sensor")
-    assert clip_presence_sensor is None
-
+    assert hass.states.get("binary_sensor.temperature_sensor") is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)